django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -93,6 +94,7 @@ class UserProfileSerializer(serializers.Serializer):
         serializers.ListField()
     )  # This is in active use by the Django frontend react components at least
     teams_full = UserTeamSerializer(many=True)
+    is_staff = serializers.BooleanField(allow_null=True)
```

Comment:
I'd argue this doesn't need to allow null but rather we just set it to False for unauthenticated users. That still makes sense for what the field represents but simplifies the field a slight bit (2 potential options rather than 3)